circle.yml

```diff
@@ -17,4 +17,4 @@ dependencies:
 test:
   override:
     - MIX_ENV=prod mix compile --warnings-as-errors
```

Comment:
Is this necessary anymore? It looks like it's done in `bin/test_suite` (though without `MIX_ENV=prod`)

Reply:
I think it's still fine, especially since this is all very quick. This assures me that it will definitely compile in the prod env, whereas bin/test_suite ensures tests pass and compile with the test env. Now we have ecto and phoenix_ecto in test only, so this will double-check that prod can compile without those deps.